packages/data/lib/src/request/tmdb_api/tmdb_crew_and_cast_images.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:data/src/request/api_request_representable.dart';
+import 'package:data/src/request/tmdb_api/constants/endpoints.dart';
+
+class TmdbCrewAndCastImages implements APIRequestRepresentable<DioMethods>{
+  final int id;
+  TmdbCrewAndCastImages._({required this.id});
+  TmdbCrewAndCastImages.byId(int id) : this._(id: id);
+
+  @override
+  String? get baseUrl => null;
```

Comment:
Here and further it just a template to compose url which I need. it can be changed in case if I change Dio on to another http client where I will use this getters